DEVELOPMENT.md

```diff
@@ -90,6 +90,30 @@ below.
 yarn run ibazel run tensorboard -- -- --logdir [LOG_DIR]
 ```
 
+#### Running TensorBoard on dev target with iBazel
+
+```shell
+(tf)$ ibazel run tensorboard:dev -- \
+--logdir gs://tensorboard-bench-logs \
+[--bind_all] \
+[--port 6008]
+```
+
+*   `ibazel`: File watcher for bazel; automatically rebuild and start the server
```

Comment:
I think ibazel is already described in the previous section. I don't think there is any need to repeat here.